docs/installing.rst

```diff
@@ -50,6 +57,8 @@ that will help you choosing the correct version of Python to use
    Do not use the pre-installed Python that comes with the OSX operating system
    which has serious bugs that can cause application crashes.
 
+Then follow the section :ref:`using-pip` below.
+
 Microsoft Windows
 =================
 
```

Comment:
Line 66 : Change <https://github.com/steko/totalopenstation/releases> to <https://github.com/totalopenstation/totalopenstation/releases>